aws-spi-pekko-http/src/main/scala/org/apache/pekko/stream/connectors/awsspi/PekkoHttpClient.scala

```diff
@@ -151,18 +153,42 @@ object PekkoHttpClient {
     else throw new RuntimeException(s"Could not parse custom content type '$contentTypeStr'.")
   }
 
+  // based on NettyNioAsyncHttpClient and ApacheHttpClient
```

Comment:
If I read the links correctly, they just highlight that the constants exist. One issue in the ASF is that we are careful about updating our LICENSE file when we borrow 3rd party code so people look for 'based on' in our code to make sure that noone has copied 3rd party code.
It might be best to remove the links in this comment.

Reply:
> If I read the links correctly, they just highlight that the constants exist. 

They just highlight that the constants exist and which are used.

> One issue in the ASF is that we are careful about updating our LICENSE file when we borrow 3rd party code so people look for 'based on' in our code to make sure that noone has copied 3rd party code.
It might be best to remove the links in this comment.

I will remove the comments, but since aws/aws-sdk-java-v2 is licensed under apacha v2 code, is that still a problem?

Reply:
We would still need to mention borrowing of code in our LICENSE and NOTICE - this is part of the terms of the Apache License. I don't believe that is code borrowing though. I think this change just uses the documented APIs of the 3rd party Amazon libs.